app/src/main/java/org/blitzortung/android/app/view/HistogramView.kt

```diff
@@ -97,7 +97,7 @@ class HistogramView @JvmOverloads constructor(
             val maximumCount = histogram.maxOrNull() ?: 0
 
             canvas.drawText(
-                "%.1f/%s _".format(
```

Comment:
That underscore marks the axis tick, which the value matches, so it should stay there.